docs/docs/lib/edge/cloudflare.mdx

```diff
@@ -0,0 +1,351 @@
+  },
+};
+```
+
+### Targeting Attributes
+
+The following targeting attributes are set automatically by the Edge App.
+
+- `id` - creates a long-lived `gbuuid` cookie if it doesn't exist already
+- `url`
+- `path`
+- `host`
+- `query`
+- `pageTitle`
+- `deviceType` - either `mobile` or `desktop`
+- `browser` - one of `chrome`, `edge`, `firefox`, `safari`, or `unknown`
+- `utmSource`
+- `utmMedium`
+- `utmCampaign`
+- `utmTerm`
+- `utmContent`
+
+You can customize both the primary identifier name (`id`) and cookie name (`gbuuid`) by setting the `UUID_KEY` and `UUID_COOKIE_NAME` environment variables respectively.
+
+As shown in the [runtime configuration](#runtime-configuration) section above, you can also pass custom attributes via runtime config. You can also skip automatic attribute generation and rely solely on custom attributes by setting the environment variable `SKIP_AUTO_ATTRIBUTES="true"`.
+
+### Routing
+
+By default, the Edge App will process all `GET` requests (other HTTP verbs are proxied through without running through our app logic).
+
+It is generally preferable to configure your routing rules outside of our Edge App. For instance, you may only want to invoke the Edge App at `https://yourdomain.io/landing-page`. You can configure CloudFlare routing by following the CloudFlare Workers [Routes <ExternalLink />](https://developers.cloudflare.com/workers/configuration/routing/routes/) documentation.
+
+There may be situations when you will need to provide finer-grained routing / URL targeting rules within our Edge App. You will need to include a JSON encoded string of route rules in your `ROUTES` environment variable.
+
+For instance, you may want to do a proxy pass-through (do not process) for `mysite.io/account/*` or `mysite.io/settings/*`. Your routes may look like this:
+
+```bash
+ROUTES='[{ "pattern":"mysite.io/account/*", "behavior":"proxy" }, { "pattern":"mysite.io/settings/*", "behavior":"proxy" }]'
+```
+
+A route uses the following interface, with many of the properties being optional:
+
+```ts
+{
+  pattern: string;
+  type?: "regex" | "simple";  // default: "simple"
+  behavior?: "intercept" | "proxy" | "error";  // default: "intercept"
+  includeFileExtensions?: boolean;  // Include requests to filenames like "*.jpg". default: false (pass-through).
+  statusCode?: number; // Alter the status code (default is 404 when using "error")
+  body?: string; // Alter the body (for setting an error message body)
+}
+```
+
+When multiple routes are included in your `ROUTES` array, only the first match is used.
+
+### Cookie Policy and GDPR
+
+By default, the Edge App will persist a random unique identifier in a first-party cookie names `gbuuid`. Its purpose is to provide a consistent user experience to your visitors by preventing them from being re-bucketed into different A/B test variations. It follows the same mechanism as discussed in our [HTML Script Tag docs](/lib/script-tag#cookie-policy-and-gdpr).
```

Comment:
```suggestion
By default, the Edge App will persist a random unique identifier in a first-party cookie named `gbuuid`. Its purpose is to provide a consistent user experience to your visitors by preventing them from being re-bucketed into different A/B test variations. It follows the same mechanism as discussed in our [HTML Script Tag docs](/lib/script-tag#cookie-policy-and-gdpr).
```

Reply:
Should we have a section here where we say anything about how to use this `gbuuid` in other tracking calls to ensure you can merge with metric data? Seems like a big part of actually getting experimentation running that we give no guidance on.

Reply:
probably best addressed by another doc/guide, since edge doesn't affect this